tests/drivers/i2s/i2s_speed/Kconfig

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2021, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "I2S Speed Test"
+
+source "Kconfig.zephyr"
+
+config I2S_TEST_SEPARATE_LOOPBACK_PORTS
+	bool "Use two separate I2S ports for loopback"
+	help
+	  Use separate I2S ports for transmit and receive.
+
+config I2S_TEST_TX_MASTER
```

Comment:
Do we need this, is this related to something specific about the NXP hardware for this test?

Reply:
This is needed as in the loopback test we are using 2 separate I2S ports, one is configured as Master and the other as Slave. The current test is written for a loopback within the same port which is not available in NXP SoC.

Reply:
Do we have any sense if we did this configuration always regardless if TX & RX are split or not what the implication would be?

Put another way, if on a STM32 or SAM device we set TX_MASTER=y and RX_MASTER=n, what do we think would happen?

Reply:
This configuration is available for all boards to use. The default behavior is for TX and RX ports to be in slave mode which is how the test currently functions. Anything different from the default can be configured via these configs.

Reply:
I guess my question was can we also have the settings as you need for NXP and thus remove the Kconfig options.

Reply:
That would mean conditional code using an NXP specific define. I felt a generic approach would be better suited as individual boards can have their own setting on MASTER and SLAVE ports.

Reply:
The fact that both functions `test_i2s_rx_transfer_configure` and `test_i2s_tx_transfer_configure` configure SPI interface pins as slave is actually a mistake. It works on Atmel SAM E70 platform since we're relying on the internal loopback but that's not a proper setup. We could modify the testcase and configure TX as master and RX as slave, have one fixed configuration.

In principle it would be good to test different configurations: TX master -> RX slave, TX slave -> RX master but we probably shouldn't introduce Kconfig options. Having user select TX master -> RX master can potentially damage hardware since we will have two pins configured as output shorted together.

Reply:
The idea behind the Kconfig was not for users to try different options. Rather it was a way to have individual developers pick the setting needed for their boards.

Reply:
Thanks, I have dropped the config options and updated the test code to configure TX as master and RX as slave.